spicy/runtime/src/driver.cc

```diff
@@ -163,7 +166,7 @@ Result<spicy::rt::ParsedUnit> Driver::processInput(const spicy::rt::Parser& pars
 
         if ( ! r ) {
             DRIVER_DEBUG(fmt("beginning parsing input (eod=%s)", data->isFrozen()));
-            r = parser.parse3(unit, data, {});
+            r = parser.parse3(unit, data, {}, {});
```

Comment:
I am probably missing something, but shouldn't we inject a context from the outside here as well? I would have expected this function to take an `optional<UnitContext>` and pass it on here.

Reply:
`processInput()` doesn't have a notion of a connection, so default behavior is fine (which means: the parser will automatically create a conetxt instance if we don't pass one in).